doc-QuickStart_OpenStack_Provider_Guide/topics/CFME_Lifecycle.adoc

```diff
@@ -0,0 +1,25 @@
+[[cfme-lifecycle]]
+== {product-title_short} Lifecycle Management
+
+This chapter discusses lifecycle activities that are part of the _{product-title}_ Automate component; these activies can include the provisioning and retirement of instances. _{product-title}_ Automate enables real-time, bidirectional process integration and adaptive automation for management events and administrative or operational activities.
```

Comment:
File removed as part of: https://github.com/ManageIQ/manageiq_docs/pull/184/commits/87068f77677f6a031edab31278c1f5272f15b5e0

Didn't need the file, I was planning to remove it in a subsequent commit.